component/pom.xml

```diff
@@ -165,23 +182,36 @@
                             org.wso2.extension.siddhi.map.avro.*
                         </Export-Package>
                         <Import-Package>
-                            com.fasterxml.jackson.databind.*,
-                            javax.annotation.*,
-                            javax.xml.*,
-                            org.apache.log4j.*,
-                            org.apache.tapestry5.json.*,
-                            org.wso2.siddhi.annotation.*,
-                            org.wso2.siddhi.core.config.*,
-                            org.wso2.siddhi.core.event.*,
-                            org.wso2.siddhi.core.exception.*,
-                            org.wso2.siddhi.core.stream..*,
-                            org.wso2.siddhi.core.util.*,
-                            org.wso2.siddhi.query.api.definition.*,
-                            org.json.*,
-                            org.slf4j.*,
-                            org.w3c.*,
-                            org.xml.sax.*,
-                            sun.misc.*,
+                            com.fasterxml.jackson.databind;resolution:=optional;vers
```

Comment:
Please identify the packages that need to be imported here correctly and change imports according to it. Please refer below[1] as a reference.

[1] - https://github.com/wso2/carbon-kernel/blob/master/core/pom.xml#L165-L180

Reply:
Refactored the pom. Changes done in commit 90d80de47aacd47dd767c01c000b0fecf6e6577d